receiver/oracledbreceiver/integration_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright Splunk, Inc.
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integration
+// +build integration
+
+package oracledbreceiver
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/consumer/consumertest"
+)
+
+// This test ensures the collector can connect to an Oracle DB, and properly get metrics. It's not intended to
+// test the receiver itself.
+func TestOracleDBIntegration(t *testing.T) {
```

Comment:
Do you mind moving this to a new `tests/oracledb` directory? https://github.com/signalfx/splunk-otel-collector/tree/main/tests/receivers

Reply:
Done, though this is not all working quite yet.